tests/pytorch_tests/function_tests/test_hessian_info_calculator.py

```diff
@@ -35,7 +36,7 @@
 class basic_model(torch.nn.Module):
     def __init__(self):
         super(basic_model, self).__init__()
-        self.conv1 = Conv2d(3, 3, kernel_size=1, stride=1)
+        self.conv1 = Conv2d(3, 3, kernel_size=3, stride=1, padding=1)
```

Comment:
why is this change necessary?

Reply:
Small kernel with small input randomly zero the gradients (I assume relu randomly gets all-negative inputs). This was the simplest "solution" to significantly decrease the chance of that happening.